pandas/core/series.py

```diff
@@ -1462,63 +1463,21 @@ def dot(self, other):
         else:  # pragma: no cover
             raise TypeError('unsupported type: %s' % type(other))
 
-    def searchsorted(self, v, side='left', sorter=None):
-        """Find indices where elements should be inserted to maintain order.
+    @Substitution(klass='Series')
+    @Appender(base._shared_docs['searchsorted'])
+    def searchsorted(self, v, side='left', *args):
+        validate_args(args, min_length=0, max_length=1)
+        sorter = None if not args else args[0]
 
-        Find the indices into a sorted Series `self` such that, if the
-        corresponding elements in `v` were inserted before the indices, the
-        order of `self` would be preserved.
+        if not isinstance(self._values, Index):
+            if sorter is not None:
+                sorter = com._ensure_platform_int(sorter)
 
```

Comment:
I agree with Joris. It's documented in the API docs:
http://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.searchsorted.html

On Mon, Mar 14, 2016 at 6:14 PM, Jeff Reback <notifications@github.com>
wrote:

> In pandas/core/series.py
> <https://github.com/pydata/pandas/pull/12413#discussion_r56102581>:
>
> >
> > -        Find the indices into a sorted Series `self` such that, if the
> > -        corresponding elements in `v` were inserted before the indices, the
> > -        order of `self` would be preserved.
> > +        if not isinstance(self._values, Index):
> > +            if sorter is not None:
> > +                sorter = com._ensure_platform_int(sorter)
> >
>
> it's a non supported argument that serves no purpose and removing it is
> not an API change that is visible - it will still be accepted just not
> advertised so I see no reason not to clean this up now
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/pydata/pandas/pull/12413/files#r56102581
>